### ISSUE TITLE
Windows deployment dependencies

### ISSUE THREAD

**Issue by caryan** [BBN-Q/libaps2#50]
Moving this [TODO](https://github.com/BBN-Q/libaps2/blob/51af0a66224661471e1cefbdc8ace85d49d03436/src/CMakeLists.txt#L153) to an issue. 

Another solution is to build with VS.

**Comment by matthewware** [comment]
Are we still building against hdf5-1.8.13?  And this would check package numbers?

**Comment by caryan** [comment]
We just build against whatever msys2's pacman gives us.  There seems to be no elegant way of handling an already loaded hdf5 dll hence the h5py/pytables continuing fiasco on Windows. This issue is more about packaging convenience similar to bring in the `make install` and `make package` we added to Linux.

**Comment by matthewware** [comment]
Hum... Orion can't seem to find the hdf5 libraries after upgrading libaps2 and hdf5.  CMake crashes and burns.  I've spent possibly too much time trying to make a package out of the older source for 1.8.13 to compile against, but if you think this isn't the issue I'll start looking other places.

**Comment by rmcgurrin** [comment]
I think I had this issue at some point.  It may have something to do with the system PATH variable.  It may be picking up the hdf5 library in the anaconda path.  Search in your anaconda root dir and you may find a conflicting library.

**Comment by rmcgurrin** [comment]
If Anaconda2/Library/bin is in your PATH you'll have a conflict
```
rmcgurrin@Qontrol MINGW64 ~/Anaconda2
$ find . -name "*.dll" | grep -i hdf
./Library/bin/hdf5.dll
./Library/bin/hdf5_cpp.dll
./Library/bin/hdf5_hl.dll
./Library/bin/hdf5_hl_cpp.dll
./Library/bin/hdf5_tools.dll

```

**Comment by caryan** [comment]
Just for the record:
*  @matthewware 's issue disappeared once Matlab was no longer holding on to the DLL 
* there should be no conflict with Anaconda's HDF5 and if there is use the `HDF5_ROOT` environment variable before running `cmake`

**Comment by caryan** [comment]
Closed by be69d37dd758b2f571a97acd7ea9a1295f89b946
